pkg/apiserver/conprof/feature_enable.go

```diff
@@ -0,0 +1,34 @@
+// Copyright 2020 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package conprof
+
+import (
+	"github.com/pingcap/tidb-dashboard/pkg/apiserver/utils"
+	"github.com/pingcap/tidb-dashboard/pkg/config"
+)
+
+var (
+	supportedTiDBVersions = []string{">= 5.3.0"}
+	featureEnabled        *bool
+)
+
+func IsFeatureEnable(config *config.Config) (enabled bool) {
```

Comment:
Considering that there are many features that require a feature flag, how about abstracting this into a util. e.g.
```
type FeatureFlag struct {
	supportedTiDBVersions []string
	featureEnabled                *bool
}

func (ff *FeatureFlag) IsFeatureEnable(targetVersion string) bool
func (ff *FeatureFlag) Middleware(targetVersion string) gin.HandlerFunc
```

And we can declare feature flags in module.go/mod.go(in the future)
```
var FeatureFlag = util.NewFeatureFlag([]string{">= 5.3.0"})
var Module = fx...
```

Reply:
LGTM, and I will do this refinement in https://github.com/pingcap/tidb-dashboard/pull/1035 .